### PR TITLE
adding gisele to team

### PR DESCRIPTION
**Pull request by beatrizcp87**
@alexbrahastoll or @oswaldoferreira can you review the english page?

Thank you

### CODE REVIEW

**Comment by beatrizcp87** [comment]
snif :cry: ninguém.

File: _posts/team-members/en/2015-06-03-gisele-torres.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+image2x: /images/time/time-gisele@2x.jpg
+full_name: Gisele Torres
+job_title: Process Anlyst
+slug: gisele
+social_accounts:
+- provider: facebook
+  username: gi.torres.3
+facebook: gi.torres.3
+bio:
+- Gisele is graduated on Company Manegement and have a MBA on People Management. ​​She knows the value of interpersonal relations and believes that harmonious coexistence should be prioritized, always!
```
Comment (marcioj) [inline _posts/team-members/en/2015-06-03-gisele-torres.html]: `Manegement` => `Management`

File: _posts/team-members/en/2015-06-03-gisele-torres.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- She was born in Aracaju and likes to quote Guimarães Rosa ( Brasilian Writer ) to describe the place taht she was raised.
```
Comment (marcioj) [inline _posts/team-members/en/2015-06-03-gisele-torres.html]: `taht` => `that`

File: _posts/team-members/en/2015-06-03-gisele-torres.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- Has a deeply love for movies. When isn't work she likes to paint and write on her blog.
```
Comment (marcioj) [inline _posts/team-members/en/2015-06-03-gisele-torres.html]: She has a deeply love for movies and when isn't working, likes to paint and write on her blog.

File: _posts/team-members/en/2015-06-03-gisele-torres.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- provider: facebook
+  username: gi.torres.3
+facebook: gi.torres.3
+bio:
+- Gisele is graduated on Company Manegement and have a MBA on People Management. ​​She knows the value of interpersonal relations and believes that harmonious coexistence should be prioritized, always!
+- She was born in Aracaju and likes to quote Guimarães Rosa ( Brasilian Writer ) to describe the place taht she was raised.
+- Has a deeply love for movies. When isn't work she likes to paint and write on her blog. 
+- She is very attentive to everything, always alert. Very thorough with everything. 
+quote:
+  content: Eu sei de muito pouco. Mas tenho a meu favor tudo o que não sei...
```
Comment (marcioj) [inline _posts/team-members/en/2015-06-03-gisele-torres.html]: Essa aqui acho que faria mais sentido pegar a versão em ingles original, ao inves de traduzir literalmente. Mas nao achei :/

**Pull request merged**